Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -888,7 +880,7 @@
   name: loadout-group-station-engineer-head
   minLimit: 0
   loadouts:
-  - StationEngineerHardhatYellow
+  #- StationEngineerHardhatYellow Moffstation - (Nox Loadout Improvements) - Yellow and blue hardhats are restricted to atmos to keep the two roles visually distinct.
```

Comment:
Related to the atmos comment, engineers can vend yellow hard hats from their drobe at roundstart, so I think this change is moot and ought not be made.